src/Storage/Factory/EntityFactory.php

```diff
@@ -44,7 +44,7 @@ public function createService(array $parameters): ?StorageInterface
         }
         $this->registered = true;
 
-         return new EntityStorage($this->doctrine);
+         return new EntityStorage($this->doctrine, $config[self::STORAGE_CONFIG_READ_ONLY], $config[self::STORAGE_CONFIG_TO_SKIP]);
```

Comment:
Small detail, I prefer "skip" in stead of "to-skip"
you use "read-only" and not "to-be-ready-only"